phpstan.neon

```diff
@@ -6,6 +6,7 @@ parameters:
 	tmpDir: ./var/cache/phpstan
 	level: 6
 	reportUnmatchedIgnoredErrors: false
+	checkGenericClassInNonGenericObjectType: false
```

Comment:
Is this disabled only because of this one issue? https://github.com/mautic/mautic/actions/runs/7125580913/job/19401784994#step:10:87

Reply:
Indeed 👍 personally I disable this until the full type coverage is reached, as it lead to focus on too detailed tiny spot and doesn't bring any value